frb_codegen/src/ir/ty.rs

```diff
@@ -127,15 +123,10 @@ impl IrType {
 #[enum_dispatch]
 pub trait IrTypeTrait {
     fn visit_children_types<F: FnMut(&IrType) -> bool>(&self, f: &mut F, ir_file: &IrFile);
-
     fn safe_ident(&self) -> String;
-
     fn dart_api_type(&self) -> String;
-
     fn dart_wire_type(&self, target: Target) -> String;
-
     fn rust_api_type(&self) -> String;
-
     fn rust_wire_type(&self, target: Target) -> String;
```

Comment:
uber-nit: seems that your formatter does not agree with previous contributor's ;) so, shall we somehow create a stricter linter rule that everyone has to follow, thus avoiding such code "change" - which will make `git blame` and code review harder

Reply:
My bad, I'll try to revert some of these nonessential changes.

Reply:
It's OK. The main problem is, we may need to somehow setup a consistent formatting rule. You know, for example, in Dart there is one consistent formatter so everyone's code is the same